spec/spec_helper.rb

```diff
@@ -0,0 +1,38 @@
+require "simplecov"
```

Comment:
Move simplecov requires and config to support/config/simplecov.rb and require it here please

Reply:
Fixed.